elastic/kibana
# [maps] remove duplicated error handling logic

### Test instructions
1. install sample web logs data set
2. create new map
3. add heatmap layer. 
4. add filter
    ```
    {
      "error_query": {
        "indices": [
          {
            "error_type": "exception",
            "message": "local shard failure message 123",
            "name": "kibana_sample_data_logs"
          }
        ]
      }
    }
    ```
5. verify error is displayed in legend and clicking "View details" button opens inspector and displays tile error
